app/scripts/services/validateRegistrant.js

```diff
@@ -59,6 +59,60 @@ angular.module('confRegistrationWebApp')
         (ruleOperand === 'AND' && validRuleCount === blockTypeSpecificRules.length);
     };
 
+    var choiceVisibleRuleCheck = function(block, choice, registrant, ruleType){
+      var answers = registrant.answers;
+      var ruleOperand = '';
+      var validRuleCount = 0;
+      var blockTypeSpecificRules = [];
+
+      if (ruleType === ruleTypeConstants.SHOW_OPTION) {
+        blockTypeSpecificRules = _.filter(block.rules, { 'ruleType': ruleType, 'blockEntityOption': choice.value });
+        ruleOperand = 'OR';
+      } else {
+        if($window.Rollbar){
+          $window.Rollbar.error('choiceVisibleRuleCheck was called with an unknown rule type: ', ruleType);
+        }
+        ruleOperand = 'AND';
```

Comment:
If the ruleType is unknown it may be better to return `false`. Or change ruleOperand to `OR` since answer rules are always `OR` and not `AND`.